sound/soc/sof/Kconfig

```diff
@@ -154,6 +154,14 @@ config SND_SOC_SOF_DEBUG_ENABLE_DEBUGFS_CACHE
 	  Say Y if you want to enable caching the memory windows.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_DEBUG_ENABLE_IPC_FLOOD_TEST
+	bool "SOF enable IPC flood test"
+	help
+	  This option enables the IPC flood test which can be used to flood
+	  the DSP with test IPCs and gather stats about response times.
+	  Say Y if you want to enable IPC flood test.
+	  If unsure, select "N".
+
```

Comment:
This is probably a wider discussion, but I do wonder why we need a separate Kconfig option for this? To me, there should be a higher bar for addition of new debug config options. If a debug feature is useful and generic enough, it should be enabled if SOF_DEBUG is enabled and otherwise disabled, unless it has a large impact to resource usage. I do note we have very fine-grained controls now, so if that's the policy and that is found good, not objecting adding one more in this patch.

Reply:
the policy for now is that nothing is selected by default. it's very hard to make decisions on behalf of distros. some willl select everything since they don't know what they are doing, others will be very picky about image size and attack surface. Those debug features are invasive and can lead to Heisenbugs.